pkg/mcs/tso/server/server.go

```diff
@@ -250,8 +428,155 @@ func checkStream(streamCtx context.Context, cancel context.CancelFunc, done chan
 }
 
 // GetTLSConfig get the security config.
-// TODO: implement it
 func (s *Server) GetTLSConfig() *grpcutil.TLSConfig {
+	return &s.cfg.Security.TLSConfig
+}
+
+func (s *Server) initClient() error {
+	tlsConfig, err := s.cfg.Security.ToTLSConfig()
+	if err != nil {
+		return err
+	}
+	u, err := types.NewURLs(strings.Split(s.cfg.BackendEndpoints, ","))
+	if err != nil {
+		return err
+	}
+	s.backendUrls = []url.URL(u)
```

Comment:
can remove `[]url.URL`

Reply:
you're right. types.urls is actually []url.URL